packages/manager/apps/telecom/src/app/telecom/pack/meetings/xdsl-meetings.routing.js

```diff
@@ -0,0 +1,64 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('telecom.xdsl-meetings', {
+    url: '/xdsl/:serviceName/meetings',
+    views: {
+      'telecomView@telecom': 'packXdslMeetings',
+    },
+    resolve: {
+      serviceName: /* @ngInject */ ($transition$) =>
+        $transition$.params().serviceName,
+      loadMeetings: /* @ngInject */ ($rootScope, OvhApiXdsl, serviceName) => {
+        return OvhApiXdsl.v6().searchOrderMeetings($rootScope, {
+          serviceName,
+        });
+      },
+      meetingSlots: /* @ngInject */ (loadMeetings) => {
+        const { result, error } = loadMeetings;
+        if (result) {
+          return {
+            slots: result.meetingSlots,
+          };
+        }
+
+        if (error) {
+          return {
+            errorMessage: error,
+          };
+        }
+        return null;
+      },
+      meetings: /* @ngInject */ (meetingSlots) => {
+        console.log('meetingSlots', meetingSlots, meetingSlots.slots);
+        const meetings = [];
+
+        let slots = [];
+        let prevTitle;
+        if (meetingSlots.slots) {
+          meetingSlots.slots.forEach((slot, index) => {
+            const title = moment(slot.startDate).format('ddd DD MMM YYYY');
+            if (!prevTitle) {
+              prevTitle = title;
+            } else if (prevTitle !== title) {
+              meetings.push({
+                title: prevTitle,
+                slots,
+              });
+              slots = [];
+              prevTitle = title;
+            }
+            slots.push({
+              id: index,
+              start: slot.startDate,
+              end: slot.endDate,
+              startTime: moment(slot.startDate).format('HH:mm'),
+              endTime: moment(slot.endDate).format('HH:mm'),
+              selected: false,
+            });
```

Comment:
forEach + push = map

Reply:
I can't do exactly the same thing with a map